### PR TITLE
V2alpha remainder

### PR DESCRIPTION
**Pull request by lyschoening**

### CODE REVIEW

File: proto/asgt/v2alpha/dataset_service.proto [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+message BatchCreateExampleRequest {
+  string dataset_name = 1;
+  repeated asgt.type.Example examples = 2;
+}
+
+message TruncateDatasetRequest {
+  string name = 1;
+}
```
Comment (michalderdak) [inline proto/asgt/v2alpha/dataset_service.proto:119]: new line

File: proto/asgt/type/example.proto [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import "asgt/type/target_value.proto";
+
+// A pair of example data and example target values
+message Example {
+  asgt.type.Data data = 1;
+  repeated asgt.type.TargetValue target_values = 2;
+  // An optional id to provide individual corrections to examples, for example when a user updates their initial
+  // feedback. The id should be a string in UUID format.
+  string id = 3;
+}
```
Comment (michalderdak) [inline proto/asgt/type/example.proto:17]: new line

File: scripts/go-gen-asgt.sh [hunk 3]
```diff
@@ -19,20 +17,31 @@ protoc -I./deps/googleapis -I./deps/tensorflow -I./deps/protoc-gen-openapiv2 -I.
 protoc -I./deps/googleapis -I./deps/tensorflow -I./deps/protoc-gen-openapiv2 -I./proto \
     --openapiv2_out gen/openapiv2/ \
     --openapiv2_opt logtostderr=true \
+    --openapiv2_opt allow_merge=true \
+    --openapiv2_opt merge_file_name=asgt/v2alpha.json \
```
Comment (michalderdak) [inline scripts/go-gen-asgt.sh:21]: what's this for?

**Review by michalderdak — DISMISSED** [review]
Looks super good! How about validation? planning to add that afterwards?

File: scripts/go-gen-asgt.sh (hunk 3 above)
Comment (lyschoening) [reply scripts/go-gen-asgt.sh:21]: @michalderdak So without this undocumented feature we get one swagger json file per proto file, which means we would need one Swagger UI per service. This merges all documentation for the selected proto in a single file.

**Comment by lyschoening** [comment]
> Looks super good! How about validation? planning to add that afterwards?

@michalderdak Yes, it seems like a separate step, since we mostly need validation on types, which are shared with the v1 API.

**Review by michalderdak — APPROVED** [review]

**Pull request merged**